### DISCUSSION TITLE
Plugins don't take effect when SSL certificate expires

### DISCUSSION THREAD

**Discussion by luozhouyang** [Kong/kong#12224] [Help]
I noticed that when a SSL certificate expires, kong plugins seems to not take effect. When I updated the SSL certificate, the same request return as expected. Is this a bug?

**Comment by StarlightIbuki** [comment]
Could you share more details of the issue so we can help?

**Comment by tobiasehlert** [comment]
@luozhouyang, are you seeing the same as me in issue #12442?
